addons/payment_adyen/models/payment_transaction.py

```diff
@@ -282,3 +283,84 @@ def _adyen_tokenize_from_feedback_data(self, data):
         _logger.info(
             "created token with id %s for partner with id %s", token.id, self.partner_id.id
         )
+
+    def _adyen_extend_api_data(self, data):
```

Comment:
The `data` parameter is unused.

Reply:
You are right. I use scheme from similar methods. The `data` might be used later or in extensions, so I'd prefer to keep it

Reply:
Please remove the `data` attribute. We can still add it later if needed (which I doubt), and we don't want to maintain a sort of API in `payment_adyen`.

Reply:
> We can still add it later 

Stable branch policy doesn't allow to do that. That's why I'd prefer to keep it